source/includes/markdown/_13-platform-ui.html.md

```diff
@@ -548,19 +548,19 @@ Once your app is submitted, an Asana developer will configure and enable your ap
 |» `x96`                      | String (url) | 96x96 icon asset |
 |» `x192`                     | String (url) | 192x192 icon asset |
```

Comment:
Somewhat unrelated, but the icons are now directly uploaded through the dev console (old or new both work) so we can probably drop these.

Reply:
Sounds good! Removing the icon object and all string URLs